app/templates/briefs/_brief_response_data.html

```diff
@@ -0,0 +1,29 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% for section in content.summary(brief_response) %}
+    {% if not (section.id == 'your-nice-to-have-skills-and-experience' and not section.get_question('niceToHaveRequirements').boolean_list_questions) %}
```

Comment:
Is there any way to avoid this?
Why do we need both conditions? I think I don't really understand what we're checking.
Is there a way to remove the section from display manifest?